api_core/setup.py

```diff
@@ -22,12 +22,12 @@
 
 name = 'google-api-core'
 description = 'Google API client core library'
-version = '1.5.0'
+version = '1.6.0a1'
```

Comment:
please don't modify setup.py in this PR. Releasetool will do it.

Reply:
Ack, will revert.